test/e2e/specs/editor/blocks/paragraph.spec.js

```diff
@@ -17,10 +17,9 @@ test.describe( 'Paragraph', () => {
 		} );
 		await page.keyboard.type( '1' );
 
-		const firstBlockTagName = await page.evaluate( () => {
-			return document.querySelector(
-				'.block-editor-block-list__layout .wp-block'
-			).tagName;
+		const firstBlock = await page.locator( 'block=*' );
```

Comment:
Nit: Let's add a `.first()` for semantic reason?

```suggestion
		const firstBlock = await page.locator( 'block=*' ).first();
```